backend/modules/load/domain/commands.py

```diff
@@ -1,9 +1,9 @@
 from dataclasses import dataclass
 
 from ...common.domain import commands as common_commands
-from ...data.domain import value_objects as data_value_objects
+from ...load.services import queries as load_queries
```

Comment:
IT MUST BE CORRECTED.

command can NEVER use anything from queries.
As `CQRS` says, command and queries must be separated.

I can see that we have `class InputData(pa.DataFrameModel):` declared in modules/data.
According to what we have now, we must do like this:

1) Move `class InputData(pa.DataFrameModel):` to models/extract/domain/value_objects
2) remove modules/data folder
3) In modules/transform/domain/value_objects at frozen data class with structure (in this case it will be one to one as in queries)
4) Use added data class in step 3 in transform commands
5) In modules/load/domain/commands use data class defined in modules/extract/domain/value_objects